e2e/internal/e2e/singularitycmd.go

```diff
@@ -411,16 +414,30 @@ func RunSingularity(t *testing.T, cmdPath string, cmdOps ...SingularityCmdOp) {
 		t.Errorf("ExpectExit is missing in cmdOps argument")
 		return
 	}
+	if s.profile == 0 {
+		availableProfiles := make([]string, len(Profiles))
+		for i, p := range Profiles {
+			availableProfiles[i] = p.String()
+		}
+		profiles := strings.Join(availableProfiles, ", ")
+		t.Errorf("you must specify a profile, available profiles are %s", profiles)
```

Comment:
So the default is no profile and leads to an error, meaning that it is assumed that a profile *must* be explicitly specified by the test?

Reply:
yes